site/_posts/2017-08-01-limericks-week-in-miq.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Limericks Week in ManageIQ"
+author: d-m-u, AllenBW
+date: 2017-08-01
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/O05SBdpl7Gc" frameborder="0" allowfullscreen></iframe>  
```

Comment:
@d-m-u - It seems this embed url is not rendering correctly - at least when I'm viewing the html from the browser.